structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,201 @@
+
+* The process must be democratic and give a voice to any community member that is otherwise eligible to join the Program's Discord server under the Collaborator role, even if they're not currently present on the server
+* Each eligible member of the community must have equal voting power
+* Eligible members may only be excluded via a reasonable inactivity threshold
+* The process must be formally documented somewhere, but this documentation may be kept internal to the given community
+* Participating communities must allow the Program's Manager team to audit their process, which may be motivated by reports from the given community's members, or from other Program members, including the Managers themselves
+
+Once a community has decided how it's going to vote, it must designate a single eligible community member to represent that vote. This representative must report their community's vote in the attached thread, specifically mentioning the community's name and the option being voted for, and providing proof of the results --- such as a screenshot of an internal poll.
+
+Participating communities also have the option of abstaining from the vote. This must either be reported in the thread as described above or, in the case of a community that wishes to abstain from all votes, directly to a Program Manager in advance. Regardless of the method, documented consensus among eligible community members must be provided.
+
+All polls must include a closing date that is set at least ten days into the future. If needed, this date may be pushed back by up to a month, as long as an informal agreement between the majority of the Program's participating, non-abstaining communities has been reached, and documented within the thread attached to the poll.
+
+At any point before the poll has closed, any Collaborator can use their right to veto, if allowed by the process. This right is described below.
+
+Once the poll has been closed, the vote should be interpreted as described by the process the poll is a part of.
+
+### The right to veto
+
+While a poll is open and accepting votes, any Collaborator or Manager can use their right to veto by sending a message to the attached thread, detailing why they'd like to veto the vote.
+
+If another Collaborator or Manager isn't satisfied with the reasoning behind the veto, they can attempt to overrule the veto. To start this process, a new poll is created to allow voters to decide whether to overrule the veto. This poll lasts for three days and, if more than half of the individual votes are positive, the veto is overruled. If not, the original vote should be treated as if it has failed.
+
+The veto process may be started on behalf of another Manager or Collaborator. Additionally, the list of voters on that veto may be kept secret. This is intended to avoid exposing those that supported the veto (or that didn't) to negative reactions or bullying by other Program members.
```

Comment:
When a veto process is started on behalf of another, are the original vetoers required to be named, or can they remain anonymous? If the latter, should there be a clause requiring their identities be transmitted in secret to Program Managers and held in confidence, for the purpose of recordkeeping?

Does "list of voters on that veto" pertain to the veto overrule poll?

Reply:
I think these are both pretty clear already via the explanation of intent? I guess I can add "started or"